package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-pay-tracking",
-  "version": "1.1.0",
+  "version": "1.0.1",
```

Comment:
I think I'd bump the version to 1.2.0 rather than 1.0.1. Reason being its a big overhaul rather than small bug fixes.